docs/collecting-data/collecting-from-own-applications/mobile-trackers/remote-configuration/index.md

```diff
@@ -203,7 +239,30 @@ Snowplow.refresh(onSuccess: successCallback)
 ```
 
   </TabItem>
-  <TabItem value="android" label="Android">
+  <TabItem value="android" label="Android (Kotlin)">
+
+```kotlin
+// Load configuration with userId = nil
```

Comment:
nit: this is incredibly nitpicky but there are two kinds of comments being used in these snippets, and no comments for the `...later...` lines so the colouring is wrong 🫠

Reply:
Fixed!